scripts/scripts.js

```diff
@@ -879,6 +938,7 @@ const helixSideKickObserver = () => {
  * @param {Element} doc The container element
  */
 async function loadLazy(doc) {
+  helixSideKickObserver();
```

Comment:
Was there a reason this was moved from the bottom of loadLazy to the top? I would keep it where it originally was (just before the sampleRUM call)

Reply:
the reason for that is there's an intermittent issue on observing sidekickready for publishing confirmation because of the await functions before setting up the sidekick observer.
![image](https://github.com/hlxsites/accenture-newsroom/assets/142872664/57f51321-23a5-4e53-bce2-939b5bedcff1)

Reply:
just making sure that the eventListeners and observer is immediately setup as the sidekick elements aren't loaded at the same time

Reply:
I would move it below site critical code even if it means a few seconds delay. we should not impact site loading performance for any authoring tool needs.

Reply:
outside of that one change i'm good with the rest of this